scripts/add_identifiers_to_existing_preprints.py

```diff
@@ -31,6 +34,30 @@ def add_identifiers_to_preprints():
     logger.info('Finished Adding identifiers to {} preprints.'.format(preprints_without_identifiers.count()))
 
 
+def main(dry=True):
+    # Start a transaction that will be rolled back if any exceptions are un
+    with transaction.atomic():
```

Comment:
This might not be a good idea...even when `--dry` is passed, a request to create the DOIs and ARKs will still be made. Also, this script will take multiple minutes, during which we'd be locking all of the preprints without identifiers.